CONTRIBUTING.md

```diff
@@ -2,6 +2,29 @@
 
 We'd love to accept your patches and contributions to this project.
 
+## Importing an AEP from google.aip.dev
+
+Currently, the project is focused on importing AIPs from
```

Comment:
```suggestion
Currently, the project is focused on adopting AIPs from
```

I think maybe "importing" implies that we aren't modifying the content at all?  But I'm not sure this word changes it all that much. WDYT?